docs: expand on a11y features of devtools

adamjohnson opened the pull request:
## What I did

1. Added / expanded on docs for Chrome/Edge/Safari/Firefox DevTools a11y features
2. Added some imagery
3. Added `h5` and `h6` to the `reset.css` to align with the rest of the heading tags.

## To Do's

- [ ] Docs for chai-a11y-axe for web-test-runner
- [ ] Docs for Playwright a11ySnapshot feature for unit tests

## Testing Instructions

1. View the QA & Testing page in the DP
2. Review for grammar
3. Review imagery (designers probably want to make edits here)
4. Does this content make sense?

## Notes to Reviewers

This page is getting a little long. Should it be broken out? If so, what should we call it? "Browser accessibility tools"?

Also, initially based this branch off of `staging/clefairy` instead of `main`. LMK if that's inhospitable. 

Closes #1644.

bennypowers commented on docs/accessibility/q-a-testing.md:
```diff
@@ -219,4 +220,96 @@ One neat feature included within Firefox's accessibility inspector is the abilit
+          await element.updateComplete;
+        });
+
+        it('should have a link with the name, "Skip to main content"', async function() {
```
can we use the chai helpers here instead of deep assertions?

see https://bennypowers.dev/decks/semantic-component-testing/#slide-25/step-3

adamjohnson replied:
Great suggestion!

Addressed in [5ed9a66](https://github.com/RedHat-UX/red-hat-design-system/pull/1903/commits/5ed9a66afdc28a0483adc8f8d3bdefc261b6f1f5). I also added some copy + a link to PFE tools to reference the custom Chai helpers to point people in the right direction. PTAL and LMK what you think.

adamjohnson commented:
The "QA & Testing" page was getting really long and needed to be broken up into multiple pages. Here's what happened:

  * "QA & Testing" content moved to other pages, "QA & Testing" page is removed
  * Added the "Accessibility Tools" page
  * Added the "Manual Testing" page
  * Added the "CI/CD" page

Related to #1913. cc/ @hellogreg @coreyvickery

hellogreg commented:
Is "CI/CD" a sufficient name for the page, or should it be "CI/CD testing" or "CI/CD tools"? (If "CI/CD" alone implies one or both of the latter, then that's fine.)

bennypowers commented:
@hellogreg CI/CD is generally understood the refer to the _practice_ of continually integrating and deploying changes to software.

so that implies the use of tools, but doesn't directly denote them. 

Personally, I think it's understood as-is.

bennypowers commented:
I think this checks all the boxes, we can polish after
